File: Herencia/ejercicio 1/main.py
from cliente import Cliente
from jefe import Jefe

c1 = Cliente("jhon", "llanque", "987654", "N001", "C44")
j1 = Jefe("nilda", "torres", "151661", "Sucursal la paz", "General")

print(c1.mostrar_datos())
print(j1.mostrar_datos())

File: Herencia/ejercicio 1/Main.java
package herencia;
public class Main {
    public static void main(String[] args) {
        Cliente c1 = new Cliente("Jhon", "llanque", "987654", "N001", "C44");
        Jefe j1 = new Jefe("Ana", "Torres", "151661", "Sucursal La paz", "General");

        System.out.println(c1.mostrarDatos());
        System.out.println(j1.mostrarDatos());
    }
}